brianlong/www.validators.app
# 181571520_check_dc_location_new

#### What's this PR do?
- Create script to change validator's data_center

#### How should this be manually tested?
- Run `bundle exec ruby script/data_centers_scripts/change_validator_data_center.rb validator_id`, check logs in `log/change_validator_data_center.log`

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181571520

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
